tests/resources/VCH-Util.robot

```diff
@@ -427,6 +427,7 @@ Cleanup VIC Appliance On Test Server
     Log To Console  Deleting the VCH appliance %{VCH-NAME}
     ${output}=  Run VIC Machine Delete Command
     Run Keyword And Ignore Error  Cleanup VCH Bridge Network  %{VCH-NAME}
+    Run Keyword And Ignore Error  Run  govc datastore.rm %{VCH-NAME}-VOL
```

Comment:
Would this mean that `vic-machine delete` failed to clean up the volume store folder?

Reply:
probably